plugins/mvn/README.md

```diff
@@ -0,0 +1,28 @@
+## The mvn Plugin
+
+The [mvn](https://github.com/robbyrussell/oh-my-zsh/tree/master/plugins/mvn) plugin provides many [aliases](#Aliases).
+
+Enable it by adding _mvn_ to the [_plugins array_](https://github.com/robbyrussell/oh-my-zsh/blob/master/templates/zshrc.zsh-template#L48) before sourcing OMZ (see [Plugins](https://github.com/robbyrussell/oh-my-zsh/wiki/Plugins)).
```

Comment:
This is too verbose, a simple "add to the plugins array" will suffice. Also, please format properly the table of aliases. [Here's a sample readme](https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/plugins/xcode/README.md) that does both these things well.